lib/gui/app/components/safe-webview.js

```diff
@@ -221,27 +220,24 @@ class SafeWebview extends react.PureComponent {
    * @example
    *
    * // In the webview
-   * console.log(JSON.stringify({
-   *   command: 'error',
-   *   data: 'Good night!'
-   * }));
-   *
-   * console.log(JSON.stringify({
-   *   command: 'log',
-   *   data: 'Hello, Mars!'
-   * }));
+   * console.log('Good night!')
    */
   static consoleMessage (event) {
-    if (!robot.isMessage(event.message)) {
+    if (_.isNil(event.message)) {
       return
     }
 
-    const message = robot.parseMessage(event.message)
+    let message = event.message
```

Comment:
Huh I thought this'd be a linter error but I guess you need `prefer-destructuring` for it to start enforcing `let { message } = event`

Reply:
I don't think we'd want to force the use of destructuring – while it's useful in some situations, it doesn't have to be used everywhere just for the sake of it.

Reply:
I don't think we should have any problems with enforcing it when we have renamings, e.g. `let { name: newName } = object`, but yeah I was just under the impression it was already on, I'm fine with it as-is.